Data-Structure-ARRAYS.md

```diff
@@ -0,0 +1,62 @@
+#Arrays
+
+`array` is a kind of data structure that can store a fixed-size sequential collection of elements of the same type. An `array` is used to store 
+a collection of data, but it is often more useful to think of an `array` as a collection of variables of the same type.
+
+`array` consists of contiguous memory locations. The lowest address corresponds to the first element and the highest address to the last 
```

Comment:
Just start with something, that clearly says this is not a necessary requirement to be an Array. As in, in C, this would be true. But at a high-level scripting language, it might not be true.